website/pages/docs/nia/installation/configuration.mdx

```diff
@@ -16,7 +16,7 @@ An example HCL configuration is shown below to automate a task to execute a Terr
 
 ```hcl
 log_level = "info"
-port = 8505
+port = 8550
```

Comment:
I would suggest removing this in case someone copy-pasta this block, unless this is the recommended port to use instead of the default 8558.